### ISSUE TITLE
//eslint-enable not working as expected

### ISSUE THREAD

**Issue by aladdin-add** [eslint/eslint#14717]
**Tell us about your environment**

* **ESLint Version:** latest
* **Node Version:** n/w
* **npm Version:** n/w
* **Operating System:** n/w 

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**
default

**What did you expect to happen?**
report an error.

**What actually happened? Please copy-paste the actual, raw output from ESLint.**
no error reported.

**Steps to reproduce this issue:**

https://eslint.org/demo#eyJ0ZXh0IjoiLyplc2xpbnQtZW5hYmxlIG5vLWRlYnVnZ2VyKi9cbmRlYnVnZ2VyOyIsIm9wdGlvbnMiOnsicGFyc2VyT3B0aW9ucyI6eyJlY21hVmVyc2lvbiI6MTIsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnt9LCJlbnYiOnt9fX0=

**Are you willing to submit a pull request to fix this bug?**
y.

**Comment by mdjermanovic** [comment]
Rule has to be configured in a config file or `eslint` comment.

`eslint-enable` just re-enables rules that were disabled with `eslint-disable`:

```js
/* eslint no-debugger: error */

debugger; // error
/*eslint-disable no-debugger*/
debugger; // no error
/*eslint-enable no-debugger*/
debugger; // error 
```

[Online Demo](https://eslint.org/demo#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLWRlYnVnZ2VyOiBlcnJvciAqL1xuXG5kZWJ1Z2dlcjsgLy8gZXJyb3Jcbi8qZXNsaW50LWRpc2FibGUgbm8tZGVidWdnZXIqL1xuZGVidWdnZXI7IC8vIG5vIGVycm9yXG4vKmVzbGludC1lbmFibGUgbm8tZGVidWdnZXIqL1xuZGVidWdnZXI7IC8vIGVycm9yIiwib3B0aW9ucyI6eyJwYXJzZXJPcHRpb25zIjp7ImVjbWFWZXJzaW9uIjoxMiwic291cmNlVHlwZSI6Im1vZHVsZSIsImVjbWFGZWF0dXJlcyI6e319LCJydWxlcyI6e30sImVudiI6e319fQ==)
